### PR TITLE
Fork and rename the project to zeek-kafka

### PR DESCRIPTION
**Pull request by JonZeolla**
## Contributor Comments
This renames the project to `zeek-kafka` from `metron-bro-plugin-kafka`.  Many of the changes in this PR are based off of other plugins, such as Johanna's [PostgreSQL](https://github.com/0xxon/zeek-postgresql) writer, and establishes some basic governance.

## TODO
- [x] Run end to end
- [x] Address [this feedback](https://github.com/SeisoLLC/zeek-kafka/pull/1#pullrequestreview-560513472)

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -1,5 +1,5 @@
-   Apache Metron
-   Copyright 2015-2020 The Apache Software Foundation
+   The Initial Developer of this project, which was copied from, derived from,
```
Comment (JonZeolla) [inline NOTICE:1]: I am not sure if what I did here was reasonable, but it was based off of [the ASF's example notice](https://infra.apache.org/licensing-howto.html#example-notice) and [this stackexchange answer](https://opensource.stackexchange.com/a/4440)

File: README.md [hunk 2]
```diff
@@ -1,8 +1,6 @@
 # Logging Zeek Output to Kafka
```
Comment (JonZeolla) [inline README.md:1]: @ottobackwards what do you think about adding a reference to Apache Metron/`metron-bro-plugin-kafka` once the December 2020 minutes are uploaded to https://www.apache.org/foundation/records/minutes/2020/?  Open to where to put it - top, middle, bottom...

**Review by ottobackwards — CHANGES_REQUESTED** [review]
This is a good start.
We need to go by https://github.com/SeisoLLC/zeek-kafka/blob/92b85e5e00cc2fd9023ea7f53466db6592eb6634/COPYING#L89

So, we need to update the NOTICE FILE,  and add the new copyright to all the files that have changed.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+zeek-kafka@seisollc.com.
```
Comment (JonZeolla) [inline .github/CONTRIBUTING.md:63]: This is copy/paste from [here](https://www.contributor-covenant.org/version/2/0/code_of_conduct/code_of_conduct.md) except for this line.

**Comment by JonZeolla** [comment]
In searching around I found [this stackexchange answer](https://opensource.stackexchange.com/a/4440). Not sure yet if this indicates any specific action we should take to update copyrights, etc.

**Comment by JonZeolla** [comment]
> In searching around I found [this stackexchange answer](https://opensource.stackexchange.com/a/4440). Not sure yet if this indicates any specific action we should take to update copyrights, etc.

Looks like https://github.com/SeisoLLC/zeek-kafka/pull/1#pullrequestreview-560513472 answers this question.  Will add to TODO

File: NOTICE (hunk 1 above)
Comment (ottobackwards) [reply NOTICE:1]: I think to top should be Zeek-Kafka
with he new copyright.
Followed by this

**Comment by JonZeolla** [comment]
e2e tests are now working.

**Comment by JonZeolla** [comment]
> > In searching around I found [this stackexchange answer](https://opensource.stackexchange.com/a/4440). Not sure yet if this indicates any specific action we should take to update copyrights, etc.
> 
> Looks like [#1 (review)](https://github.com/SeisoLLC/zeek-kafka/pull/1#pullrequestreview-560513472) addresses this statement. Will add to TODO

Read through section 4 of the license a few times and did some googling; I think e5e6990 should work, what do you think @ottobackwards?  The ASF [intentionally leaves out any copyrighting](https://www.apache.org/legal/src-headers.html#headers) from their recommended notice, but [4b](https://github.com/SeisoLLC/zeek-kafka/blob/92b85e5e00cc2fd9023ea7f53466db6592eb6634/COPYING#L97-L98) requires notification of modification, and [this open source stackexchange post](https://opensource.stackexchange.com/a/5117) seems to indicate that adding a copyright notice is sufficient.

**Comment by JonZeolla** [comment]
I'm also considering a CI step to ensure that any modified files since the last commit to hit `main` have a copyright by the zeek-kafka contributors, what do you think?

**Review by ottobackwards — COMMENTED** [review]
My only question with the copyrights is if it should have both, the new one above an old one.
IE.  where there is a copyright already, add the new one above
where there is none and we add one, put the old one below too

**Comment by ottobackwards** [comment]
@JonZeolla I think we are in agreement right?

**Review by ottobackwards — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ottobackwards) [reply README.md:1]: I think we we do that as a follow on.

**Review by ottobackwards — DISMISSED** [review]

**Comment by JonZeolla** [comment]
I was also considering an old copyright.  I say we do that before merging this if we're going to

**Comment by JonZeolla** [comment]
Ok, per our conversation I also updated the license headers to be the typical apache 2.0 now that is no longer an asf project

**Comment by ottobackwards** [comment]
Is this ready for review?

**Comment by JonZeolla** [comment]
Not yet, let me wrap up CI and double check my work.

**Comment by JonZeolla** [comment]
Ok @ottobackwards this is good for review now

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
+# Contributor Covenant Code of Conduct
+
```
Comment (ottobackwards) [inline .github/CONTRIBUTING.md:2]: Maybe we can add a section on changing the code of conduct?  like

"Code of conduct changes will be submitted openly as a PR to this document, and must be approved by ....."

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+*Where applicable, consider providing a patch that uses the [end to end testing environment](https://github.com/seisollc/zeek-kafka/blob/main/docker/run_end_to_end.sh).*
+
+## Logs, errors, etc.
+...
+
+## Your environment
+* Version of Zeek
+* Version or commit hash of the zeek-kafka package
+* Operating System and version
```
Comment (ottobackwards) [inline .github/ISSUE_TEMPLATE.md:18]: * docker version 

maybe we want system info script?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -1,26 +1,18 @@
-## Contributor Comments
-[Please place any comments here.  A description of the problem/enhancement, how to reproduce the issue, your testing methodology, etc.]
+*Thank you for submitting a contribution to the zeek-kafka writer plugin.
 
+In order to streamline the review process, we ask you follow [these guidelines](https://github.com/SeisoLLC/zeek-kafka/blob/main/.github/CONTRIBUTING.md) and complete the following template.*
 
-## Pull Request Checklist
+# Summary of the contribution
+...
```
Comment (ottobackwards) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: Versions tested with:  zeek / docker

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -1,26 +1,18 @@
...
-  ```
-  bro-pkg test $GITHUB_USERNAME/metron-bro-plugin-kafka --version $BRANCH
-  ```
-- [ ] Have you written or updated unit tests and or integration tests to verify your changes?
-- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
-- [ ] Have you verified the basic functionality of the build by building and running locally with Apache Metron's [Vagrant full-dev environment](https://github.com/apache/metron/tree/master/metron-deployment/development/centos6) or the equivalent?
+## Testing
+...
 
+## Checklist
```
Comment (ottobackwards) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: Does this change require a certain version of zeek?

File: ci/lint.sh [hunk 8]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+#  Copyright 2020-2021 Zeek-Kafka
+#
+
```
Comment (ottobackwards) [inline ci/lint.sh:5]: apache lic

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -1,26 +1,18 @@
...
+- [ ] Confirm that any associated issues are indicated in the pull request title
```
Comment (ottobackwards) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: SHELLCheck should be run and pass for all script changes. 
If a check is disabled, it should be commented as to why

**Review by ottobackwards — CHANGES_REQUESTED** [review]
some inline comments

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 6 above)
Comment (JonZeolla) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: I have a follow on that runs this in CI so I don't feel the need to duplicate manually, what do you think?  Draft PR @ #9

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 7 above)
Comment (JonZeolla) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: That should be an edit to zkg.meta, yes?  Maybe, did you ensure zkg.meta is [still] accurate

File: .github/ISSUE_TEMPLATE.md (hunk 5 above)
Comment (JonZeolla) [reply .github/ISSUE_TEMPLATE.md:18]: That makes sense but I would rather handle as a follow-on.  Can you open an issue?

File: .github/CONTRIBUTING.md (hunk 4 above)
Comment (JonZeolla) [reply .github/CONTRIBUTING.md:2]: I could go either way.  Would you mind either suggesting specific language or where to put it, or handle this as a follow-on?

**Comment by JonZeolla** [comment]
@ottobackwards feedback addressed, back to you

**Comment by JonZeolla** [comment]
GitHub support got back to me and they are opening an internal issue with the Actions team, but they expect that after merging this PR GitHub Actions will be enabled and start running for this and future commits/PRs.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 7 above)
Comment (ottobackwards) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: we can leave it for now

**Review by ottobackwards — APPROVED** [review]

**Pull request merged**